### DISCUSSION TITLE
Dark Reader makes Google Sites embedded Iframe backgrounds become white

### DISCUSSION THREAD

**Discussion by ghas0012** [darkreader/darkreader#10408] [Q&A]
Hello,

I'm a junior dev with limited frontend knowledge trying to build a portfolio using Google Sites. The main way to get any custom element in the page on their site is to use their embed feature, which gives you an iframe that runs your input HTML.

DarkReader fills the entire iframe background with a solid white color. You can find a great example on the banner on [my site](https://www.ghasemi.dev), but I saw the effect on [another google site](https://www.preesh.us/play) as well. 

I've tested, and can confirm this problem happens with an empty Iframe all the same. The problem doesn't occur if you load the page without darkreader and then turn it on.

I tried disabling/bypassing darkreader using `<meta name="darkreader-lock">`.  I also tried injecting the same meta tag up to the parent element using JS but the problem persisted.

The only thing that actually prevented darkreader's tampering altogether is `<meta name="color-scheme" content="dark">`, but that replaces the transparent background with some default dark color when darkreader is turned off, which is still undesirable. I tried adding/removing `<meta name="color-scheme" content="dark">` dynamically and it still didn't resolve the issue.

I sincerely apologize if this is badly written or the wrong place to ask, and I'd very much appreciate any direction or guidance.

**Comment by Kiansjet** [comment]
I've opened a issue #13947 for this.
